tests/php/test-class-amp-pwa-script-sanitizer.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Class AMP_PWA_Script_Sanitizer_Test.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+use AmpProject\AmpWP\ValidationExemption;
+use AmpProject\Dom\Document;
+use AmpProject\Html\Tag;
+use AmpProject\AmpWP\Tests\TestCase;
+
+/**
+ * Tests the PWA Plugin Sanitizer class.
+ *
+ * @coversDefaultClass AMP_PWA_Script_Sanitizer
+ */
+class AMP_PWA_Script_Sanitizer_Test extends TestCase {
+
+	/**
+	 * @covers::sanitize
+	 */
+	public function test_sanitize() {
```

Comment:
There should be a condition here to `markTestSkipped` if `is_offline` or `is_500` don't exist. This could happen if PWA is not present on the testing environment or isn't supported by a given version of WP.